platform/ios/scripts/package.sh

```diff
@@ -98,6 +99,17 @@ fi
 
 LIBS=(Mapbox.a)
 
+function copy_map_files {
+    MAP_FILE_ARCHS='armv7 arm64'
+    for ARCH in ${MAP_FILE_ARCHS}; do
+        MAP_FILE="${PRODUCTS}/${BUILDTYPE}-iphoneos/${NAME}-LinkMap-${VARIANT}-${ARCH}.txt"
+        if [[ -e ${MAP_FILE} ]]; then
+            step "Copying map file for ${ARCH}"
+            cp ${MAP_FILE} ${OUTPUT}/dynamic/
```

Comment:
I like to add the `-v` flag, so that file names and paths are preserved in logs.